swap/src/cli/command.rs

```diff
@@ -1303,6 +1316,10 @@ mod tests {
             self.json = true;
             self
         }
+
+        pub fn to_boxed(self) -> Box<Self> {
+            Box::new(self)
+        }
```

Comment:
Nit: Rust naming guidelines would say this should be called:


```suggestion
        pub fn into_boxed(self) -> Box<Self> {
            Box::new(self)
        }
```

Because you are consuming a non-copy argument.